config/webpack.dev.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = Merge.smart(commonConfig, {
             options: {
               sourceMap: true,
               modules: true,
-              localIdentName: '[path][name]__[local]--[hash:base64:5]',
```

Comment:
@arizzitano @edx/educator-dahlia Do we want to rename our existing scss files to be componentNameStyles.scss?  For example: `AssetsPageStyles.scss` instead of `AssetsPage (directory)` > `styles.scess` -- if so, I think we should be able to update webpack here to be `[name]__[local]` and still have the unique name.

Reply:
Actually, Now that you write it out, I sort of like that it's `style.scss`, which mimics `index.jsx` per directory/component.

Reply:
I can do that for the moment -- the idea for making it a more discrete name is to change the style name from being something like `src-AssetsPage-AssetsFilters-styles__filter-set` to `AssetsFiltersStyles__filter-set`